trio/_dtls.py

```diff
@@ -1084,9 +1148,9 @@ def get_cleartext_mtu(self):
         """
         if not self._did_handshake:
             raise trio.NeedHandshakeError
-        return self._ssl.get_cleartext_mtu()
+        return self._ssl.get_cleartext_mtu()  # type: ignore[no-any-return]
 
-    def statistics(self):
+    def statistics(self) -> DTLSChannelStatistics:
         """Returns an object with statistics about this connection.
```

Comment:
Cross-reference newly-documented statistics object instead of re-describing here.